utils/build-presets.ini

```diff
@@ -899,6 +900,10 @@ android-icu-i18n=%(arm_dir)s/libicui18nswift.so
 android-icu-i18n-include=%(arm_dir)s/icu/source/i18n
 android-icu-data=%(arm_dir)s/libicudataswift.so
 
+cross-compile-deps-path=%(prebuilt_path)
+cross-compile-build-swift-tools=0
+llvm-ninja-targets-for-cross-compile-hosts=help
```

Comment:
@edymtt, I used your new flag since I want LLVM built natively but not cross-compiled. I couldn't use the `clean` target that `--build-llvm=0` uses, because that errors for a cross-compiled LLVM. This `help` ninja target simply dumps the list of targets and does nothing.